client/data/files/types.d.ts

```diff
@@ -0,0 +1,73 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { ApiError } from 'wcpay/types/errors';
+import ACTION_TYPES from './action-types';
+
+export interface File {
+	/**
+	 * Unique identifier for the file, expected to be prefixed by file_
+	 */
+	id: string;
+	/**
+	 * 'dispute_evidence' is expected here, as this is the only type of file upload used by the API at this time.
```

Comment:
`purpose` is used for all stripe-provided files. This type is currently only used by disputes.

Reply:
I'm referring to the type `File` – can it be more specifically named (the comment on `purpose` prompted it!).

Reply:
`File` type is similar to `Dispute` or `Charge` type. IMO `FileUpload` would imply that it was only used for uploading.  
Keeping similar language and naming to other APIs made sense to me.

Reply:
Understood, I have a pet peeve about `File` because it's such a generic word (unlike `Dispute` and `Charge`). However if these are 1:1 named to match APIs that's a reasonable convention (and something to consider delegating to robots instead of humans).